### PR TITLE
Slimevr

### PR DESCRIPTION
**Pull request by ImUrX**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by JakobDev** [comment]
bot, build dev.slimevr.SlimeVR

File: dev.slimevr.SlimeVR.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        url: https://github.com/SlimeVR/SlimeVR-Server.git
+        tag: "v0.6.0"
+        commit: 4d7c931b696a3524125347ddb288b0257cb8ce56
+      # FIXME: Already made a PR for this change
+      # Patch for expecting a jar inside /app
+      - type: patch
+        path: jar.diff
+      # Patch for not checking if git is clean
+      - type: patch
+        path: clean.diff
```
Comment (JakobDev) [inline dev.slimevr.SlimeVR.yml:68]: You need to upload this patch

Comment (ImUrX) [reply dev.slimevr.SlimeVR.yml:68]: oops, i named it clean.patch instead of clean.diff

**Comment by JakobDev** [comment]
bot, build dev.slimevr.SlimeVR

File: dev.slimevr.SlimeVR.desktop [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Version=1.5
+Categories=Game;Development;GTK;
+Exec=slimevr
+Icon=slimevr
```
Comment (JakobDev) [inline dev.slimevr.SlimeVR.desktop:5]:
```suggestion
Icon=dev.slimevr.SlimeVR
```

**Comment by JakobDev** [comment]
bot, build dev.slimevr.SlimeVR

**Comment by JakobDev** [comment]
You can start the builds by yourself

**Comment by ImUrX** [comment]
Oh, thanks for telling me :P

**Comment by JakobDev** [comment]
You need to add a URL and the Releases to your Metainfo

**Comment by ImUrX** [comment]
bot, build dev.slimevr.SlimeVR

**Comment by ImUrX** [comment]
bot, build dev.slimevr.SlimeVR

**Comment by ImUrX** [comment]
yay

File: dev.slimevr.SlimeVR.yml [hunk 3]
```diff
@@ -0,0 +1,90 @@
+app-id: dev.slimevr.SlimeVR
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '42'
```
Comment (hfiguiere) [inline dev.slimevr.SlimeVR.yml:4]: current runtime is 43.

42 will EOL is April.

File: dev.slimevr.SlimeVR.yml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  # Network access
+  - --share=network
+  # Needed for graphics
+  - --share=ipc
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  # Add JRE bin on path
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --env=GDK_BACKEND=wayland
```
Comment (hfiguiere) [inline dev.slimevr.SlimeVR.yml:17]: so what happen if you only have X11?

File: dev.slimevr.SlimeVR.metainfo.xml [hunk 5]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.slimevr.SlimeVR</id>
+
+  <name>SlimeVR</name>
+  <summary>An app for facilitating full-body tracking in virtual reality</summary>
+
+  <metadata_license>MIT OR Apache-2.0</metadata_license>
```
[the remark this answers is not in the document]
Comment (ImUrX) [reply dev.slimevr.SlimeVR.metainfo.xml:8]: I'm PRing this file to the app itself where the license wanted is this. Can I keep it like this?

File: dev.slimevr.SlimeVR.yml (hunk 4 above)
Comment (ImUrX) [reply dev.slimevr.SlimeVR.yml:17]: It should run, doesn't GTK fallback to X11?

File: dev.slimevr.SlimeVR.yml (hunk 3 above)
Comment (ImUrX) [reply dev.slimevr.SlimeVR.yml:4]: I need to build a old version of WebKitGTK2 then

Comment (ImUrX) [reply dev.slimevr.SlimeVR.yml:4]: It's kind of hard to do this. Do you have any guides on how I could do it? It has a lot of dependencies :sob:

File: dev.slimevr.SlimeVR.yml (hunk 4 above)
Comment (ImUrX) [reply dev.slimevr.SlimeVR.yml:17]: it doesnt run in wayland by default for some reason if I dont put that

Comment (ImUrX) [reply dev.slimevr.SlimeVR.yml]: now it should work

File: dev.slimevr.SlimeVR.yml (hunk 3 above)
Comment (ImUrX) [reply dev.slimevr.SlimeVR.yml:4]: Will change it when the new tauri version gets released

**Comment by ImUrX** [comment]
bot, build dev.slimevr.SlimeVR

**Comment by ImUrX** [comment]
bot, build dev.slimevr.SlimeVR

**Comment by ImUrX** [comment]
bot, build dev.slimevr.SlimeVR

**Comment by ImUrX** [comment]
seems to work very well, had some people test it ^-^

File: dev.slimevr.SlimeVR.metainfo.xml [hunk 6]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline dev.slimevr.SlimeVR.metainfo.xml:1]: This is not a blocker, but ideally you should maintain this file in your repo.

File: dev.slimevr.SlimeVR.desktop [hunk 7]
```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
```
Comment (barthalion) [inline dev.slimevr.SlimeVR.desktop:1]: Same about this one.

**Comment by ImUrX** [comment]
Yes, I already made a PR for it :P

**Pull request closed without merging**